documentation/modules/oauth/con-oauth-reauthentication.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth.adoc
+
+[id='con-oauth-reauthentication-{context}']
+= Session re-authentication mechanism
+
+The Kafka _SASL OAUTHBEARER_ mechanism supports a so called 're-authentication' mechanism.
```

Comment:
```suggestion
The Kafka _SASL OAUTHBEARER_ mechanism supports a Kafka feature called the _re-authentication_ mechanism.
```

Reply:
_Add a new sentence to the same paragraph:_

OAuth 2.0 authentication in Strimzi provides support for the re-authentication mechanism.